pkg/kapp/cmd/app/deploy.go

```diff
@@ -448,6 +457,72 @@ func (o *DeployOptions) calculateAndPresentChanges(existingResources,
 	return clusterChangeSet, clusterChangesGraph, (len(clusterChanges) == 0), changesSummary, err
 }
 
+func CheckForPriodicAnnotation(existingResources, newResources []ctlres.Resource) error {
+
```

Comment:
nit: you can remove the empty line here and everywhere else we usually don't have an empty line after function